Makefile

```diff
@@ -67,7 +67,8 @@ lint:
 	for PKG in $(PACKAGES); do golint -set_exit_status $$PKG || exit 1; done;
 
 .PHONY: test
-test:
+test: TAGS=bindata sqlite
```

Comment:
For the record, I used to add a new target like `xtest:` specifically for the new testing facility but `go test` would still look into the directory and keep doing its thing. Another approach to prevent it is to deliberately exclude the `/tests/` directory from the `PACKAGE` variable for the `xtest` target.

Any ideas?